backend/src/api/game.rs

```diff
@@ -0,0 +1,59 @@
+use actix_web::error::{ErrorInternalServerError, ErrorNotFound};
```

Comment:
I think this should live in `api/game/challenge.rs` so that it reflects the endpoint's path